arcgis-ios-sdk-samples/Utility network/Create load report/CreateLoadReportViewController.swift

```diff
@@ -0,0 +1,357 @@
+// Copyright 2021 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class CreateLoadReportViewController: UIViewController {
+    // MARK: Storyboard views
+    
+    /// The table view to display the load reports.
+    @IBOutlet var tableView: UITableView!
+    /// The button to reset the phase summaries.
+    @IBOutlet var resetBarButtonItem: UIBarButtonItem!
+    /// The button to run the trace and create a load report.
+    @IBOutlet var runBarButtonItem: UIBarButtonItem!
+    
+    // MARK: Properties
+    
+    /// A feature service of an electric utility network in Naperville, Illinois.
+    let utilityNetwork = AGSUtilityNetwork(url: URL(string: "https://sampleserver7.arcgisonline.com/arcgis/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer")!)
```

Comment:
I'll probably need to use the new `AGSUtilityNetwork(url:map:)` initializer here... 🤔

Reply:
No there is no map.